tests/test_exporters.py

```diff
@@ -341,19 +341,20 @@ def test_nested_item(self):
         i2 = dict(name=u'bar', age=i1)
         i3 = TestItem(name=u'buz', age=i2)
 
-        self.assertExportResult(i3,
+        self.assertExportResult(
+            i3,
             b'<?xml version="1.0" encoding="utf-8"?>\n'
             b'<items>'
-                b'<item>'
```

Comment:
any strong opinion here?

Not sure if you would prefer to put some of them in the same line or even exclude them from the flake8 rule.

Reply:
I'd leave the indentation in this case, although it seems like an adjustment would be needed in the last occurrence to make it multiple of 4.

Reply:
In the testcases I don't care as much about the style. I'm okay without the indentation since I won't usually have a reason to comprehend this xml structure.
But from a readability perspective, it'd make a lot more sense to me if you rewrote the longer single-line xml strings earlier on, instead of this already reasonably readable one. This again makes it look more like a change to appease a dumb script, instead of helping some developer to grasp the code better and faster.

Reply:
I would either leave it as is or switch to more complex code that does not compromise readability (e.g. defining the document as a single, multi-line, indented string and have it parsed and serialized without spaces, as desired here).